config/initializers/mailjet.rb

```diff
@@ -0,0 +1,7 @@
+if Rails.env.production?
+  Mailjet.configure do |config|
+    config.api_key = ENV['MAILJET_API_KEY']
+    config.secret_key = ENV['MAILJET_SECRET_KEY']
```

Comment:
This app is on Heroku, so no need to do that. I've updated Heroku with the necessary env vars.